feat(guide): jdupes improve descriptions (#2035)
TRaSH-Guides/Guides: docs/Hardlinks/Replace-copies-with-hardlinks.md

> You recently switched to a proper setup that supports Hardlinks and Instant Moves (Atomic-Moves).

Reviewer: ```suggestion
Have you recently switched to a setup that supports hardlinks and Instant Moves (Atomic-Moves), and would like to replace duplicate files with hardlinks?
```

> And you would like to replace copies with hardlinks?

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```
